operator/manifests/kiali-upstream/1.10.0/kiali.v1.10.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,596 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kiali-operator.v1.10.0
+  annotations:
+    categories: Monitoring,Logging & Tracing
+    certified: "false"
+    containerImage: quay.io/kiali/kiali-operator:v1.10.0
+    capabilities: Deep Insights
+    support: Kiali
+    description: "Kiali project provides answers to the questions: What microservices are part of my Istio service mesh and how are they connected?"
+    repository: https://github.com/kiali/kiali
+    createdAt: 2019-10-11T00:00:00Z
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "kiali.io/v1alpha1",
+          "kind": "Kiali",
+          "metadata": {
+            "name": "kiali"
+          },
+          "spec": {
+            "installation_tag": "My Kiali",
+            "istio_namespace": "istio-system",
+            "deployment": {
+              "namespace": "istio-system",
```

Comment:
It looks like you have exact copies of these manifest files in both upstream and community. But you cannot copy the CSV yaml file from one to the other - they are actually slightly different. Take a look here:

https://github.com/operator-framework/community-operators/blob/master/upstream-community-operators/kiali/1.9.1/kiali.v1.9.1.clusterserviceversion.yaml#L27

versus here:

https://github.com/operator-framework/community-operators/blob/master/community-operators/kiali/1.9.1/kiali.v1.9.1.clusterserviceversion.yaml#L27

Notice the namespace is different. This is because when we place these files in a PR over in the operator framework/OLM github repo (for publication to OperatorHub.io), those PRs trigger travis tests. The travis tests for one is run on Kubernetes (the upstream manifest) and the travis tests for the other is run on OpenShift (the community manifest). And with namespace set to "istio-system", the tests fail on the Kubernetes environment (it has to go in the "default" namespace). But we can't use "default" for this as an OpenShift example because Kiali should not go in OpenShift's default namespace. And when these PR fail, the Kiali manifest will not get published to OperatorHub.io or in the OpenShift quay repo.

So you want this line to be:

```suggestion
              "namespace": "default",
```

I don't know if there are any other changes between the community and upstream manifests (I can't think of any - do a diff and see). But this is why I never copy files from upstream manifest to community manifest (or the other way around). Note: the CRD and the package files are the same, but the CSV yaml files are not.

(Hopefully you can now see why working with the operator/OLM stuff is a pain :-) This is also why I am meticulous in formatting and sorting the yaml in a consistent way - because I'm constantly having to look at files for tiny differences and having everything sorted and formatted consistently helps my eyes find things.)

Reply:
Fixed...

> Hopefully you can now see why working with the operator/OLM stuff is a pain :-)

Well... What can I add? :)
I expected a better experience.